web/themes/webspark/renovation/includes/theme.inc

```diff
@@ -146,3 +147,89 @@ function renovation_preprocess_media(array &$variables) {
     unset($attributes['data-round']);
   }
 }
+
+
+/**
+ * Implements hook_preprocess_maintenance_page().
+ */
+
+function renovation_preprocess_maintenance_page(&$variables) {
+
+  $block_manager = \Drupal::service('plugin.manager.block');
+  
+  // Building header block and props, no navtree
+
+  $header_block = \Drupal\block\Entity\Block::load('asubrandheader');
+
+  // AsuBrandHeaderBlock
+
+  if ($header_block) {
+    $settings = $header_block->get('settings');
+    $header_built = $block_manager->createInstance('asubrandheader', $settings);
+    // Some blocks might implement access check.
+    $header_render =  $header_built->build();
+    // Add the cache tags/contexts.
+    \Drupal::service('renderer')->addCacheableDependency($header_render, $header_built);  
+
+    $header_config = $header_block->get('settings');
+
+    // why do I need to rebuild this here? I can't figure out a way to access the props object created in modules/webspark/asu_brand/src/Plugin/Block/AsuBrandHeaderBlock.php directly. Doing it here does give me the opportunity to set navtree to null though.
```

Comment:
@mlsamuelson I left a long comment here, feedback welcome

Reply:
Yes, that props object is a mapping of values from the block settings into the prop object.  The long term fix for this would be to create a helper function (drupal service?) that takes the settings from the block, and returns a props object. That could then be used both in the block code you referenced and here.

3 options for moving forward with that:
1. Update your comment here and add a TODO to this effect for future improvement.
2. Create a ticket for future improvement... OR...
3. Do it! create that helper function in the `asu_brand` module - there's already a services class for helper functions for that module, so you could add the new method there: https://github.com/ASUWebPlatforms/webspark-ci/blob/master/web/modules/webspark/asu_brand/src/AsuBrandHelperFunctions.php